Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -501,11 +501,15 @@ WarpXParticleContainer::DepositCharge (WarpXParIter& pti, RealVector& wp,
 }
 
 void
-WarpXParticleContainer::DepositCharge (Vector<std::unique_ptr<MultiFab> >& rho, bool local)
+WarpXParticleContainer::DepositCharge (Vector<std::unique_ptr<MultiFab> >& rho,
+                                        bool local, bool reset)
 {
     // Loop over the refinement levels
     int const finest_level = rho.size() - 1;
-    for (int lev = 0; lev < finest_level; ++lev) {
+    for (int lev = 0; lev <= finest_level; ++lev) {
```

Comment:
The switch from `<` to `<=` in the above code is in fact correcting a typo from #520